docs/source/raft_configuration.md

```diff
@@ -143,6 +143,10 @@ the consensus section in the `orderer.yaml`:
   certificate), the node halts its operation for that channel. A node suspects
   its channel eviction when it doesn't know about any elected leader nor can be
   elected as leader in the channel. Defaults to 10 minutes.
+  * `TickIntervalOverride`: If set, the value here will be preferred over the
+  tick interval configured in all channels this orderer consents in.  This value
+  should be set only with great care, as a mismatch in tick interval across
+  orderers could result in a loss of quorum for one or more channels.
```

Comment:
Maybe better as:

If set, this value will be preferred over the tick interval configured in all channels where this ordering node is a consenter.

Reply:
Done